templates/deny.txt.j2

```diff
@@ -0,0 +1,3 @@
+{% for node_id in golem_source_deny_nodes %}
```

Comment:
Consider using {%- ... %} so there won't be empty newline

Reply:
What empty newline? This creates a nice file with every node_id in a separate file. AFAIK your solution would join them all in a single line.

Reply:
Jinja2 will put newlines in lines where {% ... %} is.

Reply:
Take an example:
In [1]: import jinja2

In [2]: tpl = """{% for node_id in golem_source_deny_nodes %}
   ...: {{ node_id }}
   ...: {% endfor %}
   ...: """

In [3]: golem_source_deny_nodes = ['one', 'two', 'three', 'four']

In [4]: t = jinja2.Template(tpl)

In [5]: t.render(golem_source_deny_nodes=golem_source_deny_nodes)
Out[5]: '\none\n\ntwo\n\nthree\n\nfour\n'

In [6]: print(t.render(golem_source_deny_nodes=golem_source_deny_nodes))

one

two

three

four

Reply:
And this is how it behaves with {%-

In [24]: tpl = """{%- for node_id in golem_source_deny_nodes %}
    ...: {{ node_id }}
    ...: {%- endfor %}
    ...: """

In [25]: t = jinja2.Template(tpl)

In [26]: t.render(golem_source_deny_nodes=golem_source_deny_nodes)
Out[26]: '\none\ntwo\nthree\nfour'

In [27]: print(t.render(golem_source_deny_nodes=golem_source_deny_nodes))

one
two
three
four

Reply:
The docs says it's set to 'no', but that's not true.
https://github.com/ansible/ansible/pull/40312

Reply:
ok, I will approve this then